client/src/contexts/user.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext, createContext } from "react";
+import useAuth from "../hooks/useAuth";
+
+const UserContext = createContext(null);
+
+export const useUserContext = function () {
+  const user = useContext(UserContext);
+  return user;
+};
```

Comment:
Any reason to do this vs export the context itself?

Reply:
It just saves you a line of code where you don't have to import useContext and the context object itself. You just import useUserContext and its there for you.

Reply:
Gotcha, that makes sense to me